src/data/faq.json

```diff
@@ -129,7 +129,7 @@
                         "anchor": "vac_cert_why",
                         "active": true,
                         "textblock": [
-                            "At the beginning of the year, the European Council decided to launch an interoperable and standardized proof of vaccination for medical purposes. Germany is now quickly implementing this European decision in order to be able to provide all citizens who wish to do so with a digital proof of vaccination. In the meantime, the European Commission has also presented a draft regulation regulating an EU-wide recognition framework for (digital) certificates for vaccinations, tests and for persons with COVID-19 recovered status in order to facilitate the free movement of travelers. The EU's green certificate forms the legal framework for the member states' solutions."
+                            "At the beginning of the year, the European Council decided to launch an interoperable and standardized proof of vaccination for medical purposes. Germany is now quickly implementing this European decision in order to be able to provide all citizens who wish to do so with a digital proof of vaccination. In the meantime, the European Commission has also presented a draft regulation regulating an EU-wide recognition framework for (digital) certificates for vaccinations, tests and for persons with COVID-19 recovered status in order to facilitate the free movement of travelers. The EU's COVID-19  certificate forms the legal framework for the member states' solutions."

```

Comment:
It seems that it is now called "EU Digital COVID Certificate" (not COVID-19) - see https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_en

Reply:
thanks